src/plugins/orangehrmBuzzPlugin/Api/BuzzPostAPI.php

```diff
@@ -420,7 +420,11 @@ public function getValidationRuleForGetOne(): ParamRuleCollection
      *         @OA\JsonContent(
      *             @OA\Property(
      *                 property="data",
-     *                 ref="#/components/schemas/Buzz-PostModel"
+     *                 type="array",
+     *                 @OA\Items(oneOf={
+     *                     @OA\Schema(ref="#/components/schemas/Buzz-PostModel"),
+     *			           @OA\Schema(ref="#/components/schemas/Buzz-FeedPostModel"),
+     *                 })
```

Comment:
```suggestion
     *                 oneOf={
     *                     @OA\Schema(ref="#/components/schemas/Buzz-PostModel"),
     *			    @OA\Schema(ref="#/components/schemas/Buzz-FeedPostModel"),
     *                 })
```
refer: `src/plugins/orangehrmBuzzPlugin/Api/BuzzCommentAPI.php`